src/components/Hero/index.tsx

```diff
@@ -13,32 +13,41 @@ export const HeroSection: FunctionComponent<IntroProps> = ({
   title,
   heading,
 }) => {
-  const [startAnimation, setAnimation] = useState(false)
-  const [startSecondAnimation, setSecondAnimation] = useState(false)
-  const [startThirdAnimation, setThirdAnimation] = useState(false)
+  const [showHeadline, setShowHeadline] = useState(false)
+  const [showSubheadline, setShowSubheadline] = useState(false)
+  const [showBackground, setShowBackground] = useState(false)
+  const [showMoreSection, setShowMoreSection] = useState(false)
 
   useEffect(() => {
-    setAnimation(true)
     setTimeout(() => {
-      setThirdAnimation(true)
+      setShowBackground(true)
```

Comment:
I tried using the `setShowBackground(true)` outside the `setTimeout()` but that does not work and the animations are not triggered. The component is rather rendered  in the final state and no transitions occure.